params/params.go

```diff
@@ -139,33 +139,38 @@ type VerifyTokenRequest struct {
 	Macaroons         macaroon.Slice `httprequest:",body"`
 }
 
-// UserSSHKeysRequest is a request for the list of ssh keys associated
+// GetSSHKeysRequest is a request for the list of ssh keys associated
 // with the specified user.
-type UserSSHKeysRequest struct {
+type GetSSHKeysRequest struct {
```

Comment:
SSHKeysRequest? (to be consistent with SSHKeysResponse).

Most of the other GET requests don't have a "Get" prefix.